pysrc/juliacall/__init__.py

```diff
@@ -248,8 +248,34 @@ def jlstr(x):
                 "PYTHON_JULIACALL_HANDLE_SIGNALS=no."
             )
 
-init()
+    # Next, automatically load the juliacall extension if we are in IPython or Jupyter
+    CONFIG['autoload_ipython_extension'] = choice('autoload_ipython_extension', ['yes', 'no'])[0]
+    if CONFIG['autoload_ipython_extension'] in {'yes', None}:
+        try:
+            get_ipython = sys.modules['IPython'].get_ipython
+
+            if CONFIG['autoload_ipython_extension'] is None:
+                # Only let the user know if it was not explicitly set
+                print(
+                    "Detected IPython. Loading juliacall extension. To disable, you can either "
```

Comment:
I see what you mean about this being quite long.

Most of this should be in the docs so please add it there - there's already a config bit in the JuliaCall page.

Then you could replace most of this message with a hyperlink to the docs?

Reply:
Done!